client/stylesheets/shared/_reset.scss

```diff
@@ -64,95 +66,42 @@
 	display: none;
 }
 
-.wp-responsive-open {
-	.woocommerce-layout__header {
-		margin-left: 2px;
-	}
-
-	@include breakpoint( '<782px' ) {
-		.woocommerce-layout__activity-panel-wrapper.is-open,
-		.woocommerce-layout__activity-panel-tabs {
-			width: calc(100vw - 190px);
+.woocommerce_page_wc-admin {
+	.wp-responsive-open {
+		.woocommerce-layout__header {
+			margin-left: 2px;
 		}
 	}
 }
 
 @include breakpoint( '<600px' ) {
-	#wpadminbar,
-	.jetpack-masterbar #wpadminbar {
+	#wpadminbar {
 		position: fixed;
 	}
 }
 
-#adminmenuwrap {
-	z-index: 99992;
+.auto-fold #adminmenu,
+.auto-fold #adminmenuback,
+.auto-fold #adminmenuwrap {
+	z-index: 99992; /* higher than woocommerce-layout__header (99991) */
```

Comment:
Is it possible to fix this by lowering the `z-index` of `.woocommerce-layout__header` instead? There are reports of the sticky nav interfering with other UI elements as well: https://wordpress.org/support/topic/terrible-plugin-doesnt-play-with-woocommerce-partial-orders-or-publishpress/#post-11634003

![](https://i.snag.gy/QdXEjp.jpg)

Reply:
Sounds like a good solution! I decreased it here: 1cd15f56e701d3169dfed9c11038e6e0a806ea32.

I had to overwrite the z-index of `components-popover` to avoid this:
![Peek 2019-06-14 16-47](https://user-images.githubusercontent.com/3616980/59517635-47274180-8ec4-11e9-98ed-d0c7831685c2.gif)